talk/owt/sdk/p2p/p2ppeerconnectionchannel.cc

```diff
@@ -1156,17 +1156,18 @@ void P2PPeerConnectionChannel::DrainPendingStreams() {
 void P2PPeerConnectionChannel::SendStop(
     std::function<void()> on_success,
     std::function<void(std::unique_ptr<Exception>)> on_failure) {
-  RTC_LOG(LS_INFO) << "Send stop.";
-  Json::Value json;
-  json[kMessageTypeKey] = kChatClosed;
-  SendSignalingMessage(json, on_success, on_failure);
+  // RTC_LOG(LS_INFO) << "Send stop.";
```

Comment:
why removing them?

Reply:
Thanks for pointing out. This one should be recovered. It's commented out when I was debugging a JSON issue (https://github.com/open-webrtc-toolkit/owt-deps-webrtc/commit/aecb781135683a142900d9a12b8e606286e22ab5).